src/plugins/run/__tests__/__fixtures__/images/unwrap.md

```diff
@@ -0,0 +1,10 @@
+Image in a paragraph: ![should NOT unwrap][ecologi_image].
+
+![should unwrap][ecologi_image]
+
+[![should unwrap][ecologi_image]][ecologi]
+
+![should NOT unwrap][ecologi_image] [![should NOT unwrap][ecologi_image]][ecologi]
```

Comment:
This is the other change - don't unwrap a paragraph of images/links if there is more than one image/link inside of it (helps support github readme badge link images that are rendered inline in a single paragraph, etc).